strategy-contract.yaml

```
@@ -0,0 +1,78 @@
+openapi: 3.0.1
+info:
+  title: Strategy Module
+  description: This is the Strategy Module of AEGEE-Europe
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.0.1
+paths:
+  /actionagenda/{aaId}:
+    get:
+      tags:
+      - Action Agenda
+      summary: Display a specified Action Agenda
+      description: For valid response try integer IDs with value >= 1 and <= 10.         Other
+        values will generated exceptions
+      operationId: getActionAgendaById
+      parameters:
+      - name: aaId
+        in: path
+        description: ID of Action Agenda that needs to be fetched
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: Get specified Action Agenda
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionAgenda'
+        400:
+          description: Invalid Action Agenda ID supplied
+          content: {}
+        404:
+          description: Action Agenda not found
+          content: {}
```

```suggestion
          content: {}
        default:
          description: Error     
```
This fixes the 401 request by Rik. I don't think we need to specify here the meaning of 401, because the whole API is only accessible with authN